### ISSUE TITLE
Exploit failed

### ISSUE THREAD

**Issue by Clemdu63** [iabem97/houdini-public#27]
When I had install houdini, I launch it and I have this error inside the app : 

```
Exploit failed
Hold the power + home button for 6 seconds to reboot then try to start again
```

**Comment by TravisBrown01** [comment]
Well how are you installing it

**Comment by Clemdu63** [comment]
Like everyone I take the ipa and I install it With cydia impactor

**Comment by WhyIsItReal** [comment]
you need to compile and sign it with a paid developer account

**Comment by Clemdu63** [comment]
Yes but when I open it on Xcode I have like 16 errors so I can’t compile anything

**Comment by WhyIsItReal** [comment]
errors or warnings? as long as you correctly sign it and build it for a real connected device it should build fine. you don't need an ipa at all

**Comment by Clemdu63** [comment]
I Will see that tonight thanks

**Comment by Clemdu63** [comment]
I had install houdini without an ipa but with Xcode and I still have the same error message

**Comment by quinl** [comment]
It can take a few tries.

**Comment by Clemdu63** [comment]
I tried 20 times so...

**Comment by Clemdu63** [comment]
Someone has an idea ?

**Comment by TravisBrown01** [comment]
Same

**Comment by Clemdu63** [comment]
I tried everything ! Even Many version of xcode and Also with macos Mojave

**Comment by TravisBrown01** [comment]
You have a paid developer account right

**Comment by Clemdu63** [comment]
No but I had remove the multipath entitlement like it’s say

**Comment by TravisBrown01** [comment]
And that’s your issue you only remove the multipath entitlement if you are running iOS 10-11.1.2 otherwise you leave it in and your required to use a developer ID

**Comment by Clemdu63** [comment]
I’m on iOS 11.3.1 and I removed multipath like it’s say on instructions

**Comment by Vanquished79** [comment]
If you are on 11.3.1 then you HAVE to have a developer account,  you only remove the multipath if you’re on a lower version like 11.1.2

**Comment by 23805130** [comment]
รับ Outlook for iOS
	





On Fri, Jun 8, 2018 at 3:20 PM +0700, "Clemdu63" <notifications@github.com> wrote:












I’m on iOS 11.3.1 and I removed multipath like it’s say on instructions

**Comment by lirui758** [comment]
别问了，11.3需要账户    而账户需要金钱

**Comment by Yashkigura** [comment]
i need 11.4 jailbreaker ipa THANKS GUYS :))

**Comment by hlmoore** [comment]
@Yashkigura that doesnt exist yet
